srcpkgs/python3-SQLAlchemy/template

```diff
@@ -1,19 +1,18 @@
 # Template file for 'python3-SQLAlchemy'
 pkgname=python3-SQLAlchemy
-version=1.3.18
-revision=6
+version=1.4.50
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 makedepends="python3-devel"
-depends="python3"
+depends="python3 python3-typing_extensions python3-greenlet"
```

Comment:
why typing-extensions?

Reply:
https://github.com/sqlalchemy/sqlalchemy/blob/rel_1_4_50/setup.cfg#L85

Reply:
that's an optional dependency if using aiosqlite

Reply:
So I `typing-extensions` along with `greenlet` should be removed ?